react-native-reanimated.d.ts

```diff
@@ -371,6 +392,29 @@ declare module 'react-native-reanimated' {
       config: DecayConfig,
     ): BackwardCompatibleWrapper;
 
+    // reanimated2 animations
+    export function withTiming(
+      toValue: number,
+      userConfig?: Omit<TimingConfig, 'toValue'>,
+      callback?: (isCancelled: boolean) => void,
+    ): number;
+    export function withSpring(
+      toValue: number,
+      userConfig?: Omit<SpringConfig, 'toValue'>,
```

Comment:
SpringConfig is missing `velocity: number`

Reply:
Done in 5fd666ea7702ff1a9a7a6f10d78b0fe3aa768521